src/components/ApplyFiltersModal/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { useQuery } from "@apollo/client";
+import { css } from "@emotion/react";
+import ConfirmationModal from "@leafygreen-ui/confirmation-modal";
+import { Body } from "@leafygreen-ui/typography";
+import { zIndex } from "constants/tokens";
+import {
+  DefaultFiltersForProjectQuery,
+  DefaultFiltersForProjectQueryVariables,
+} from "gql/generated/types";
+import { DEFAULT_FILTERS_FOR_PROJECT } from "gql/queries";
+import { useCachedTask } from "hooks/useCachedTask";
+import { useFilterParam } from "hooks/useFilterParam";
+import DefaultFilter from "./DefaultFilter";
+
+interface ApplyFiltersModalProps {
+  open: boolean;
+  setOpen: (open: boolean) => void;
+}
+
+const ApplyFiltersModal: React.FC<ApplyFiltersModalProps> = ({
+  open,
+  setOpen,
+}) => {
+  const [filters] = useFilterParam();
+
+  const task = useCachedTask();
+  const { versionMetadata } = task ?? {};
+  const { projectIdentifier = "" } = versionMetadata ?? {};
+
+  const { data } = useQuery<
+    DefaultFiltersForProjectQuery,
+    DefaultFiltersForProjectQueryVariables
+  >(DEFAULT_FILTERS_FOR_PROJECT, {
+    variables: { projectIdentifier },
+    skip: !projectIdentifier,
+  });
+  const { project } = data || {};
+  const { parsleyFilters } = project || {};
+
+  return (
+    <ConfirmationModal
+      buttonText="Apply filters"
+      css={css`
+        z-index: ${zIndex.modal};
+      `}
+      onCancel={() => setOpen(false)}
+      onConfirm={() => setOpen(false)}
+      open={open}
+      setOpen={setOpen}
+      title="Default Filters"
+    >
+      {parsleyFilters?.map((f) => (
```

Comment:
I think this would benefit from a scrollable area. (I'm not really sure what the Leafygreen convention is for overflowing modals) But I would imagine this section would grow pretty quickly as teams adopt this feature. (It will surely be really popular)